--- node_modules/irc/lib/irc.js.orig	2023-04-27 13:38:45.988423798 -0500
+++ node_modules/irc/lib/irc.js	2023-04-27 13:39:11.655867157 -0500
@@ -1151,6 +1151,7 @@
                         case 'DEPTH_ZERO_SELF_SIGNED_CERT':
                         case 'UNABLE_TO_VERIFY_LEAF_SIGNATURE':
                         case 'SELF_SIGNED_CERT_IN_CHAIN':
+                        case 'ERR_TLS_CERT_ALTNAME_INVALID':
                             if (!this.opt.selfSigned) {
                                 return tlscon.destroy(tlscon.authorizationError);
                             }
@@ -1609,4 +1610,4 @@
 function randomInt(length) {
     return Math.floor(Math.random() * length);
 }
-//# sourceMappingURL=irc.js.map
\ No newline at end of file
+//# sourceMappingURL=irc.js.map
